beacon-chain/sync/rpc_send_request.go

```diff
@@ -512,6 +520,16 @@ func SendDataColumnSidecarsByRangeRequest(
 	}
 	defer closeStream(stream, log)
 
+	requestedSlot, err := isSidecarSlotRequested(request)
+	if err != nil {
+		return nil, errors.Wrap(err, "is sidecar slot within bounds")
+	}
+	vfs = append([]DataColumnResponseValidation{
+		areSidecarsOrdered(),
+		isSidecarIndexRequested(request),
```

Comment:
Wouln't be a good idea to merge `isSidecarIndexRequested` and `isSidecarSlotRequested` in the same validator?